tests/integration/templates/opensearch_domain.yml

```diff
@@ -13,3 +13,7 @@ Resources:
         InstanceType: 'm3.medium.search'
         # test using string types instead of int
         InstanceCount: "2"
+
+Outputs:
+  DomainEndpoint:
+    Value: !GetAtt Domain.DomainEndpoint
```

Comment:
Note: this block is not required in the equivalent elasticsearch test since that test works successfully _and_ references the domain endpoint, though I don't yet know why.